templates/components/cart/totals.html

```diff
@@ -11,7 +11,7 @@
             {{/or}}
         </div>
     </li>
-    {{#if cart.gift_wrapping_cost.amount}}
```

Comment:
can you please clarify why we need changes here?

Reply:
@yurytut1993 because the "gift_wrapping_cost" object that comes to the Cart page does not contain the "amount" key, so we will not see the "Gift wrapping" display and its price.